demo/rerank_demo_docs.py

```diff
@@ -0,0 +1,20 @@
+from rank_llm.retrieve_and_rerank import retrieve_and_rerank

```

Comment:
import sys
import os
SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
sys.path.append(os.path.dirname(SCRIPT_DIR))

Reply:
This is needed so that `python demo/rerank_....py` runs properly.